client/app/kubernetes/nodes/kubernetes-nodes.html

```diff
@@ -0,0 +1,40 @@
+<div data-ng-if="$ctrl.loading" class="text-center">
+    <oui-spinner data-size="m"></oui-spinner>
```

Comment:
Sometimes `l`, sometimes `m`, is this justified ? You are supposed to use `m` if you are not loading a whole page

Reply:
I have now put `m` sizes everywhere